metrics_proto.go

```diff
@@ -39,6 +39,8 @@ import (
 	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
 	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
 	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
+	"sort"
```

Comment:
Please put these imports in the same section as the standard library imports.
i.e.
```go
import (
	"context"
	"errors"
	"fmt"
	"path"
        "sort"
        "strings"